float64 velocity
float32 steering_angle